processor/processor.go

```diff
@@ -1959,6 +1993,36 @@ func (proc *Handle) transformations(partition string, in *transformationMessage)
 	wg := sync.WaitGroup{}
 	wg.Add(len(in.groupedEvents))
 
+	spans := make([]stats.TraceSpan, 0, len(in.groupedEvents))
+	defer func() {
+		for _, span := range spans {
+			span.End()
+		}
+	}()
+
+	traces := make(map[string]stats.Tags)
+	for _, eventList := range in.groupedEvents {
+		for _, event := range eventList {
+			if event.Metadata.TraceParent == "" {
+				proc.logger.Warnf("Missing traceParent in transformations for jobId %d", event.Metadata.JobID)
+				continue
+			}
+			if _, ok := traces[event.Metadata.TraceParent]; ok {
+				continue
+			}
+			tags := stats.Tags{
+				"sourceId":      event.Metadata.SourceID,
+				"destinationId": event.Metadata.DestinationID,
+				"workspaceId":   event.Metadata.WorkspaceID,
```

Comment:
Given that above here in the processor we also included the jobid and the uuid, it might make sense to include them here as well. Wdyt?

Reply:
We read from the DB already so it's definitely not a consumer at this stage. We're going to store the message though so perhaps it could either be internal or producer.

Reply:
As discussed, removing `jobid` and the `uuid` from processor.